src/popup/components/auth/Login.vue

```diff
@@ -36,7 +36,7 @@
         >
             <ol>
                 <li>
-                    Head to <ext-link to="https://viblo.asia/settings/oauth">
+                    Head to <ext-link to="https://accounts.viblo.asia/security/tokens">
                     <strong class="text-primary">API keys</strong></ext-link> for your Viblo account.
```

Comment:
We also change this label to:
```suggestion
                    <strong class="text-primary">Personal Access Tokens
</strong></ext-link> for your Viblo account.
```